macros.in

```diff
@@ -382,6 +382,21 @@ Supplements:   (%{name} = %{version}-%{release} and langpacks-%{1})\
 #%_source_payload	w9.gzdio
 %_binary_payload	%[ %_rpmformat >=6 ? "w19.zstdio" : "w9.gzdio" ]
 
+#	Algorithm to use for generating the payload digest and alternate
+#	(uncompressed) payload digest.
+#	WARNING: non-SHA256 is backwards incompatible with rpm < 6.0!
+#	The supported algorithms may depend on the underlying crypto
+#	implementation but generally at least the following are supported:
+#	1	MD5
+#	2	SHA1
```

Comment:
Given that this is net-new, should restrictions be added to enforce nothing weaker than sha256 be used?